src/assets/custom/css/_sass/_training-hero.scss

```diff
@@ -0,0 +1,150 @@
+.training-hero__background-image {
+  background-size: cover;
+  opacity: 0.9;
+  position: relative;
+  width: 100%;
+
+  @include small {
+    height: 216.27px;
+  }
+
+  @include medium-and-large {
+    height: 433px;
+  }
+
+  @include extra-large {
+    height: 519.5px;
+  }
+}
+
+.trainint-hero__course-instructor-avatar {
```

Comment:
Is this a typo? :D (Also in the liquid file this class is used)

Reply:
good spot! thanks it is a type 😱